src/main/java/fruitshop/db/FruitShopStorage.java

```diff
@@ -0,0 +1,9 @@
+package fruitshop.db;
+
+import fruitshop.model.FruitTransaction;
+import java.util.LinkedList;
+import java.util.List;
+
+public class FruitShopStorage {
+    public static final List<FruitTransaction> fruitTransactions = new LinkedList<>();
```

Comment:
why do you use linked list here?

Reply:
Because it more suitable in this situation. It's more resizeable. And I don't access elements by index a lot

Reply:
ok, let it be